app/controllers/ping_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PingController < ActionController::Base
+  def show
+    render json: version_info
+  end
+
+  private
+
+  def version_info
+    {
+      build_date: ENV.fetch('APP_BUILD_DATE'),
```

Comment:
@lostie it's a commonly used across projects and is created as part of the build process. I cant find any documentation though - have requested info from platforms. Will update the commit message when they respond.